app/src/main/java/com/infomaniak/mail/data/models/correspondent/Recipient.kt

```diff
@@ -33,17 +34,29 @@ open class Recipient : EmbeddedRealmObject, Correspondent {
     override var email: String = ""
     override var name: String = ""
 
+    //region UI data (Transient & Ignore)
```

Comment:
Rename all `UI data` regions in the project the same way : `//region UI data (Transient & Ignore)`